test/metabase/notification/test_util.clj

```diff
@@ -193,6 +194,16 @@
                   :subject "Welcome {{payload.event_info.object.first_name}} to {{context.site_name}}"
                   :body    "Hello {{payload.event_info.object.first_name}}! Welcome to {{context.site_name}}!"}})
 
+(def default-email-handler
+  (delay {:channel_type :channel/email
```

Comment:
needs delay for `mt/user->id`

Reply:
Maybe it makes sense to put `default-slack-handler` behind delay just to make those similar?